src/scss/includes/app.scss

```diff
@@ -108,95 +108,52 @@ BODY {
   }
 }
 
-.map_container .poi-marker {
-  width: 36px;
-  height: 42px;
-  background: transparent;
-  position: relative;
+.map_container .marker-container {
   display: flex;
+  align-items: center;
   justify-content: center;
-  pointer-events: none;
-  transform: translateY(4px);
-  cursor: pointer;
-}
-
-.map_container .poi-hoverable:hover,
-.map_container .poi-hoverable.active {
-  z-index: 1;
+  width: 35px;
+  height: 42px;
+  background-image: url('/statics/images/map/pin_map.svg');
```

Comment:
we need to use relative URLs in css files, to handle potential prefix used by the deployed app (/maps/...)

Reply:
whoops, i forgot that.., i'll fix it